management/server/core/hub-manager/hub-manager-impl/src/main/java/io/subutai/core/hubmanager/impl/environment/state/build/ConfigureContainerStateHandler.java

```diff
@@ -153,6 +216,25 @@ private void removeKeys( EnvironmentId envId, Set<String> sshKeys, String[] curr
     }
 
 
+    public String getAuthorizedKeysFromContainer( ContainerHost containerHost )
```

Comment:
You could have used io.subutai.common.peer.Peer#getContainerAuthorizedKeys
here